### PR TITLE
Update BBE examples

### PR DESCRIPTION
**Pull request by pubudu91**

### CODE REVIEW

**Review by anupama-pathirage — APPROVED** [review]

File: examples/any-type/any_type.bal [hunk 1]
```diff
@@ -11,7 +11,7 @@ function main(string... args) {
     any a = 5;
     io:println(a);
 
-    // First, the variable of the `any` type needs to be casted to the required type (`int` in this example) as shown here. More information about type casting is available in the next section.
+    // First, the variable of the `any` type needs to be cast to the required type (`int` in this example) as shown here. More information about type casting is available in the next section.
```
Comment (hasithaa) [inline examples/any-type/any_type.bal]: @samgnaniah Pls review this.

Comment (samgnaniah) [reply examples/any-type/any_type.bal]: Looks good, but what do we mean by 'next section' here? When BBE is rendered to ballerina.io, we don't have sections. Shall we remove that line?

Comment (pubudu91) [reply examples/any-type/any_type.bal]: Here, it seems to refer to the next example, which, in the site, happens to be the type conversion example. Agreed, referring to a "next section" sounds weird in the context of BBE. Will explicitly mentioning the type conversion example be good enough or should we get rid of that sentence altogether?

File: examples/fork-join-variable-access/fork_join_variable_access.description [hunk 2]
```diff
@@ -1,10 +1,10 @@
-// Workers within a `fork-join` can access the variables that are declared within the scope of that statement.
+// Workers within a fork-join can access the variables that are declared within the scope of that statement.
 // The variables can be value type variables or reference type variables.
 // Value type variables, such as string, integer, float, and blob type variables, are passed into the workers as a copy
```
Comment (samgnaniah) [inline examples/fork-join-variable-access/fork_join_variable_access.description]: Shall we use backticks for type variable names, etc.?

File: examples/fork-join-variable-access/fork_join_variable_access.description [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 // of the original data. Therefore, you are able to update the value of it within the worker and that value is only
 // defined within the specific worker.
 // Reference type variables, such as map, json, xml, and records type variables, are passed into these workers as a
 // reference to the original data. Therefore, if you update the value of these variables within a worker, the value gets
 // updated for the entire function.
-// The `Join` block receives a map of `any` type values, which contains the data sent from the respective workers,
+// The Join block receives a map of `any` type values, which contains the data sent from the respective workers,
```
Comment (samgnaniah) [inline examples/fork-join-variable-access/fork_join_variable_access.description]: I think 'Join' can be in lowercase here.

File: examples/functions/functions.bal [hunk 4]
```diff
@@ -5,7 +5,7 @@ function printValue(string value) {
     io:println(value);
 }
 
-// This function takes two int values and return their sum as an integer.
+// This function takes two int values and returns their sum as an integer.
```
Comment (samgnaniah) [inline examples/functions/functions.bal]: Shall we have 'int' between backticks? I notice we have backticks in other places so let's be consistent and add backticks for all instances.

File: examples/maps/maps.bal [hunk 5]
```diff
@@ -21,20 +21,20 @@ function main(string... args) {
...
     // Print the number of keys in the map.
     io:println(lengthof addrMap);
 
-    // You can remove a key using the 'remove' method.
+    // You can remove a key using the `remove()` function.
     var isRemoved = addrMap.remove("postalCode");
     io:println(addrMap);
 
-    // Constrained maps can only contain values of type which the map is constrained to.
+    // Constrained maps can only contain values of the specified constrained type.
```
Comment (samgnaniah) [inline examples/maps/maps.bal]: 'specified constrained type' sounds unusual to me. Shall we say something like 'Constrained maps can only contain values of the specific types that are constrained." WDYT?

Comment (pubudu91) [reply examples/maps/maps.bal]: How about "Constrained maps can only contain values of the type specified by the type descriptor"? Type descriptor is the part between "<>"

**Comment by samgnaniah** [comment]
Looks good to merge.

**Pull request merged**